spec.html

```diff
@@ -787,68 +787,98 @@ <h1>Throw an Exception</h1>
 
       <emu-clause id="sec-returnifabrupt" aoid="ReturnIfAbrupt">
         <h1>ReturnIfAbrupt</h1>
-        <p>Algorithms steps that say or are otherwise equivalent to:</p>
+        <p>An algorithm step that says:</p>
         <emu-alg>
-          1. ReturnIfAbrupt(_argument_).
+          1. *[before]* ReturnIfAbrupt(_argument_) *[after]*
         </emu-alg>
-        <p>mean the same thing as:</p>
+        <p>is equivalent to the sequence of steps:</p>
         <emu-alg>
           1. If _argument_ is an abrupt completion, return _argument_.
           1. Else if _argument_ is a Completion Record, set _argument_ to _argument_.[[Value]].
+          1. *[before]* _argument_ *[after]*
         </emu-alg>
-        <p>Algorithms steps that say or are otherwise equivalent to:</p>
+        <p>where</p>
+        <ul>
+          <li>*[before]*, if present, is any algorithm text that does not contain ReturnIfAbrupt.
+          <li>*[after]*, if present, is any algorithm text.
+          <li>_argument_ is any alias.
+        </ul>
+
+        <p>An algorithm step that says:</p>
         <emu-alg>
-          1. ReturnIfAbrupt(AbstractOperation()).
+          1. *[before]* ReturnIfAbrupt(AbstractOperation(*[arguments]*)) *[after]*
         </emu-alg>
-        <p>mean the same thing as:</p>
+        <p>is equivalent to the sequence of steps:</p>
         <emu-alg>
-          1. Let _hygienicTemp_ be AbstractOperation().
+          1. Let _hygienicTemp_ be AbstractOperation(*[arguments]*).
           1. If _hygienicTemp_ is an abrupt completion, return _hygienicTemp_.
           1. Else if _hygienicTemp_ is a Completion Record, set _hygienicTemp_ to _hygienicTemp_.[[Value]].
+          1. *[before]* _hygienicTemp_ *[after]*
         </emu-alg>
-        <p>Where _hygienicTemp_ is ephemeral and visible only in the steps pertaining to ReturnIfAbrupt.</p>
-        <p>Algorithms steps that say or are otherwise equivalent to:</p>
+        <p>where</p>
+        <ul>
+          <li>*[before]*, if present, is any algorithm text that does not contain ReturnIfAbrupt.
```

Comment:
```suggestion
          <li>*[before]*, if present, is any algorithm text that does not contain ReturnIfAbrupt.</li>
```
and similarly for the other 7 `<li>` in this PR. So far, the spec always uses closing tags when they exist. (Not sure why I didn't spot this in my earlier review.)